### ISSUE TITLE
wrong version on plugin.xml, sticked on 1.1.7

### ISSUE THREAD

**Issue by elbidone** [pushandplay/cordova-plugin-apprate#121]
Hi, 

on plugin.xml the version remains 1.1.7 instead of 1.1.12

```xml
<plugin xmlns="http://cordova.apache.org/ns/plugins/1.0"
        id="cordova-plugin-apprate"
        version="1.1.7">
```

It's annoying because it appears on logs `Installing "cordova-plugin-apprate" at "1.1.7" for ios` and we could spend hours trying to find out why.

**Comment by StefanoGitHub** [comment]
+1 👍

**Comment by westonganger** [comment]
Fixed in master will be released in v1.2.0
